tracing-attributes/src/lib.rs

```diff
@@ -294,6 +305,17 @@ mod expand;
 ///     Ok(())
 /// }
 /// ```
+/// 
+/// If you're returning a `Result<T, E>` and `E` implements [`std::error::Error`], you can
+/// record the raw error value using `err(Raw)`. This can be useful when paired with a subscriber
```

Comment:
similarly, I might prefer:
```suggestion
/// If you're returning a `Result<T, E>` and `E` implements [`std::error::Error`], it can be
/// recorded the raw error value using `err(Raw)`. This can be useful when paired with a subscriber
```